Social/Facebook/Oauth.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+/*
+ * This file is part of the Integrated package.
+ *
+ * (c) e-Active B.V. <integrated@e-active.nl>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Integrated\Bundle\SocialBundle\Social\Facebook;
+
+use Facebook\Exceptions\FacebookResponseException;
+use Facebook\Exceptions\FacebookSDKException;
+use Facebook\Facebook;
+use Integrated\Bundle\SocialBundle\Oauth\OauthInterface;
+use Integrated\Common\Channel\Connector\Config\OptionsInterface;
+use Symfony\Component\HttpFoundation\RequestStack;
+
+/**
+ * Class Oauth
+ * @package Integrated\Bundle\SocialBundle\Social\Facebook
+ */
+class Oauth implements OauthInterface
+{
+    private $facebook_app_id;
+
+    private $facebook_app_id_secret;
+
+    /**
+     * @var RequestStack
+     */
+    private $requestStack;
```

Comment:
The docblock says this is a request stack but the construtor says it actualy a Request object.